modules/tinymce/changelog.txt

```diff
@@ -24,7 +24,9 @@ Version 5.5.0 (TBD)
     Changed `imagetools` context menu icon for accessing the image dialog to use the `image` icon #TINY-4141
     Changed the `editor.insertContent()` and `editor.selection.setContent()` APIs to retain leading and trailing whitespace #TINY-5966
     Changed the `table` plugin `Column` menu to include the cut, copy and paste column menu items #TINY-6374
+    Changed the default table styles in the content CSS files to better support the styling options available in the tables dialog #TINY-6179
```

Comment:
This is vague enough as a changelog item that it's still technically correct even after we removed all the stuff from the last PR. However, should we change this line anyway so that it can be differentiated from the changelog item we'll add when we do get around to merging the rest of the PR?

Reply:
Hm. I think (partly for expediency's sake) this is okay. I don't think we mind much if multiple changelog entries are similar, especially for something like this.

Reply:
I'd say instead it should be the "fix" that maybe applied later that should include a different changelog anyways :shrug: